src/api/utils/classification.py

```diff
@@ -0,0 +1,181 @@
+import datetime
+import hashlib
+import os
+import uuid
+from enum import Enum
+from io import BytesIO
+from typing import Any, Dict, Optional
+
+import pytz
+from django.contrib.gis.geos import Point
+from django.core.files.base import ContentFile
+from django.db.models.fields.files import ImageFieldFile
+from exif import Image as ExifImage
+from PIL import ExifTags, Image as PILImage
+from PIL.ExifTags import TAGS
+
+from ..models import Image
+
+
+def create_unique_image_name(image: ImageFieldFile) -> str:
+    image_name = image.name
+    image_ext = os.path.splitext(image_name)[1]
+    current_utc_time = datetime.datetime.now(datetime.timezone.utc)
+    uid = str(uuid.uuid4()).replace("-", "")
```

Comment:
Why not tie this uuid to that of the `Image` primary key? And do we need to add timestamp?